tests/integration/lifecycle/test_simple.py

```diff
@@ -40,6 +40,25 @@
 def test_actions_simple(new_dir, mocker):
```

Comment:
Perhaps break this test up?  Less simple now :D

Reply:
And it will get much more complex soon.